### ISSUE TITLE
visualize model results with rmarkdown dashboard

### ISSUE THREAD

**Issue by jsadler2** [USGS-R/drb-do-ml#41]
We need some code to visualize the model results. I think it would be pretty nice to have a little rmarkdown dashboard template that we could just give the model name to and it would render the dashboard with plots and summary information.

**Comment by jsadler2** [comment]
It seems like this would be not too difficult with the "flexdashboard" package.

**Comment by lekoenig** [comment]
That's a great idea, and thanks for linking to "flexdashboard"! Do we have a good sense of the information (certain plots, statistics, etc) we would want to include in our dashboard? I'd have to read the flexdashboard documentation further, but I'm curious how this compares with using "shiny" as @galengorski showed us for his NO3 modeling work.

**Comment by jsadler2** [comment]
Good questions. 
> Do we have a good sense of the information (certain plots, statistics, etc)

Here's what I was thinking:

Tables/text:
* overall summary stats for training and validation sets (NSE, RMSE) as table
* site-specific summary stats for training and validation sets (NSE, RMSE) as table
* model configuration/training parameters
    * number of epochs
    * learning rate
    * model name or id (i.e., `0_baseline_LSTM`)
    * input variables
    * training, validation dates
    * dropout
    * early stopping criteria
    * hidden size
    
Plots:
* Time series of train/validation predictions/observations at each site
* Pred vs. obs scatter plots at each site
* Maybe a map of locations showing performance (maybe coloring by NSE or RMSE)?

> I'm curious how this compares with using "shiny" as @galengorski showed us for his NO3 modeling work.

From my very minimal understanding of the two, I think the "flexdashboard" looks simpler to spin up. I also like that it generates an html page which we could possibly host through github pages. From what I understand with shiny, the shiny app needs to be run locally or on a webserver which makes it a little more complicated to share.

**Comment by jsadler2** [comment]
That Weights and Biases video is sweet, @galengorski. Looks like it could be really useful. I'm interested to hear what you learn from your meeting with them.

**Comment by lekoenig** [comment]
This issue wasn't flagged as a priority for FY22, so I'm closing this issue for now. We can revisit down the line if time allows.
